src/addons/ReactDOMFactories.js

```diff
@@ -0,0 +1,169 @@
+  'kbd': createDOMFactory('kbd'),
+  'keygen': createDOMFactory('keygen'),
+  'label': createDOMFactory('label'),
+  'legend': createDOMFactory('legend'),
+  'li': createDOMFactory('li'),
+  'link': createDOMFactory('link'),
+  'main': createDOMFactory('main'),
+  'map': createDOMFactory('map'),
+  'mark': createDOMFactory('mark'),
+  'menu': createDOMFactory('menu'),
+  'menuitem': createDOMFactory('menuitem'),
+  'meta': createDOMFactory('meta'),
+  'meter': createDOMFactory('meter'),
+  'nav': createDOMFactory('nav'),
+  'noscript': createDOMFactory('noscript'),
+  'object': createDOMFactory('object'),
+  'ol': createDOMFactory('ol'),
+  'optgroup': createDOMFactory('optgroup'),
+  'option': createDOMFactory('option'),
+  'output': createDOMFactory('output'),
+  'p': createDOMFactory('p'),
+  'param': createDOMFactory('param'),
+  'picture': createDOMFactory('picture'),
+  'pre': createDOMFactory('pre'),
+  'progress': createDOMFactory('progress'),
+  'q': createDOMFactory('q'),
+  'rp': createDOMFactory('rp'),
+  'rt': createDOMFactory('rt'),
+  'ruby': createDOMFactory('ruby'),
+  's': createDOMFactory('s'),
+  'samp': createDOMFactory('samp'),
+  'script': createDOMFactory('script'),
+  'section': createDOMFactory('section'),
+  'select': createDOMFactory('select'),
+  'small': createDOMFactory('small'),
+  'source': createDOMFactory('source'),
+  'span': createDOMFactory('span'),
+  'strong': createDOMFactory('strong'),
+  'style': createDOMFactory('style'),
+  'sub': createDOMFactory('sub'),
+  'summary': createDOMFactory('summary'),
+  'sup': createDOMFactory('sup'),
+  'table': createDOMFactory('table'),
+  'tbody': createDOMFactory('tbody'),
+  'td': createDOMFactory('td'),
+  'textarea': createDOMFactory('textarea'),
+  'tfoot': createDOMFactory('tfoot'),
+  'th': createDOMFactory('th'),
+  'thead': createDOMFactory('thead'),
+  'time': createDOMFactory('time'),
+  'title': createDOMFactory('title'),
+  'tr': createDOMFactory('tr'),
+  'track': createDOMFactory('track'),
+  'u': createDOMFactory('u'),
+  'ul': createDOMFactory('ul'),
+  'var': createDOMFactory('var'),
+  'video': createDOMFactory('video'),
+  'wbr': createDOMFactory('wbr'),
+
+  // SVG
+  circle: createDOMFactory('circle'),
+  'clipPath': createDOMFactory('clipPath'),
+  'defs': createDOMFactory('defs'),
+  'ellipse': createDOMFactory('ellipse'),
+  'g': createDOMFactory('g'),
+  'image': createDOMFactory('image'),
+  'line': createDOMFactory('line'),
+  'linearGradient': createDOMFactory('linearGradient'),
+  'mask': createDOMFactory('mask'),
+  'path': createDOMFactory('path'),
+  'pattern': createDOMFactory('pattern'),
+  'polygon': createDOMFactory('polygon'),
+  'polyline': createDOMFactory('polyline'),
+  'radialGradient': createDOMFactory('radialGradient'),
+  'rect': createDOMFactory('rect'),
+  'stop': createDOMFactory('stop'),
+  'svg': createDOMFactory('svg'),
+  'text': createDOMFactory('text'),
+  'tspan': createDOMFactory('tspan'),
```

Comment:
I think I read somewhere that React is getting into the habit of wrapping keys that shouldn't get modified with quotes. I went ahead and did it.